src/main/java/dk/bec/polonez/reservationsystem/dto/userDto/UpdateRequest.java

```diff
@@ -0,0 +1,24 @@
+package dk.bec.polonez.reservationsystem.dto.userDto;
```

Comment:
**advice** Incorrect package name (upper case should not be used according to Java conventions):

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

"Naming Conventions
Package names are written in all lower case to avoid conflict with the names of classes or interfaces."

Reply:
It will be corrected at the end on main branch.